ISHPullUp/ISHPullUpViewController.m

```diff
@@ -602,6 +611,10 @@ - (void)updateLayoutOfDimmingView:(UIView *)dimmingView bottomHeight:(CGFloat)bo
 }
 
 - (void)handleDimmingViewTapGesture:(UITapGestureRecognizer *)tap {
+    if (self.isLocked) {
```

Comment:
I thought this would allow other gestures to take priority over this disabled gesture